openedx/core/djangoapps/credit/api/provider.py

```diff
@@ -51,6 +50,35 @@ def get_credit_providers():
     return CreditProvider.get_credit_providers()
 
 
+def get_credit_provider_instructions(username, course_id):
+    """Retrieve the 'fulfillment_instructions' for selected credit provider
+    of a user.
+
+    Args:
+        username (str): The user initiating the request
+        course_id (str): The identifier for the course
+
+    Returns: Text content of 'fulfillment_instructions' for 'CreditProvider'
+        model.
+    """
+    credit_attribute = None
+    for attribute in get_enrollment_attributes(username, course_id):
+        if attribute['namespace'] == 'credit':
+            credit_attribute = attribute
+            break
```

Comment:
I think you should check `attribute['name'] == 'provider_id'` here instead of checking down as in future we might have more than one enrollment attributes with `namespace='credit'`